client/src/views/users/UserContextMenu.vue

```diff
@@ -3,7 +3,7 @@
 <template>
   <ion-content id="user-context-menu">
     <ion-list class="menu-list">
-      <ion-item-group class="group">
+      <ion-item-group class="group" v-if="userType === UserType.Active">
```

Comment:
It's probably better not to update the context menu at all, since the whole revoked user page is only temporary anyway.